components/atom/panel/src/index.scss

```diff
@@ -1,5 +1,6 @@
 @import '~@s-ui/theme/lib/index';
 
+$bdrs-atom-panel: 0 !default;
```

Comment:
`$bdrs-atom-panel: $bdrs-none`

Reply:
@s-ui/theme/lib/settings/_box-style.scss

Reply:
I moved that to a component prop so now I left the default value to `$bdrs-l` so in the demo we can see a difference between a floating panel or a non-floating panel